lib/scsi_defs.h

```diff
@@ -1,10 +1,13 @@
 #ifndef LIB_SCSI_DEFS_H
 #define LIB_SCSI_DEFS_H
 
+#include "absl/base/attributes.h"
 #include <cstdint>
```

Comment:
Standard library headers [should come first and be separated by an empty line](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).

Reply:
Good catch! I'll make sure to be more careful with the includes next time!